BLL/Services/SkillKnowledgeService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using AutoMapper;
+using BLL.DTO;
+using BLL.Interfaces;
+using DAL.Models;
+using DAL.Repositories;
+using DAL.Repositories.Specifications;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BLL.Services
+{
+    public class SkillKnowledgeService : ISkillKnowledgeService
+    {
+        private readonly IUnitOfWork _db;
+        private readonly IMapper _mapper;
+        public SkillKnowledgeService(IUnitOfWork db, IMapper mapper)
+        {
+            _db = db;
+            _mapper = mapper;
+        }
+        public void AddSkillKnowledge(IEnumerable<SkillKnowledgeDTO> skillKnowledgeDtoList)
+        {
+            foreach (var skillKnowledgeDto in skillKnowledgeDtoList)
```

Comment:
bad practice. Save into database in the loop

Reply:
we can remake this in future